virt/virt-4-11-release-notes.adoc

```diff
@@ -83,7 +83,7 @@ The SVVP Certification applies to:
 //CNV-18319 UI can switch between BIOS and UEFI
 * You can set the xref:../virt/virtual_machines/virt-edit-vms.adoc#virt-editing-vm-web_virt-edit-vms[boot mode] of templates and virtual machines to *BIOS*, *UEFI*, or *UEFI (secure)* by using the web console.
 
-//CNV-17269 Enable and disabe VM descheduler from UI
+* You can now enable and disable the descheduler from the *Scheduling* tab of the *Review and create VirtualMachine* step when xref:../virt/virtual_machines/virt-create-vms.adoc#virt-quick-creating-vm_virt-create-vms[creating a custom virtual machine] in the web console.
```

Comment:
I am looking at a QE test environment and there is a **Descheduler** field in the **Scheduling** tab on the **VirtualMachine details** page. Is that what you are supposed to refer to? Also, the xref points to the "Quick creating a virtual machine" module. Based on the link text, should the xref instead point to the `Creating a virtual machine from a customized template` module or the `Virtual machine fields` reference module?